vault/resource_mount_test.go

```diff
@@ -107,6 +107,26 @@ func TestResourceMount_SealWrap(t *testing.T) {
 	})
 }
 
+// Test Audit HMAC Request Keys attributes
+
+func TestResourceMount_AuditNonHMACRequestKeys(t *testing.T) {
+	path := "example-" + acctest.RandString(10)
+	resource.Test(t, resource.TestCase{
+		Providers: testProviders,
+		PreCheck:  func() { testAccPreCheck(t) },
```

Comment:
`testAccPreCheck()` was just replaced by `testutil.TestAccPreCheck()` in 5660a057265b1eb2c00c7c2015de01979caa42f6, you'll want to rebase.
```suggestion
		PreCheck:  func() { testutil.TestAccPreCheck(t) },
```